tests/test_models/test_place.py

```diff
@@ -195,6 +196,6 @@ def test_to_dict_values(self):
 
     def test_str(self):
         """test that the str method has the correct output"""
-        place = Place()
-        string = "[Place] ({}) {}".format(place.id, place.__dict__)
-        self.assertEqual(string, str(place))
+        p = Place()
+        string = "[Place] ({}) {}".format(p.id, p.__dict__)
+        self.assertEqual(string, str(p))
```

Comment:
Many of the same issues as City. Rinse and repeat.

Reply:
Check!